Resources/doc/extensions.md

```diff
@@ -218,3 +219,66 @@ With the Responsive class you can set some `display` options to define how the h
 | display  | array or null       | null    | Define how the hidden information should be displayed to the end user. |
 
 ___
+
+## 3. Select
+
+**Be sure to install the [Select Extension](https://datatables.net/extensions/select/) before using.**
+
+### Template
+
+SgDatatablesBundle:datatable:extensions.html.twig
+
+### Initialisation
+
+#### The easiest way
+
+The easiest way is to add `select` to your extensions options with a boolean value.
+
+``` php
+public function buildDatatable(array $options = array())
+{
+    // ...
+
+    $this->extensions->set(array(
+        'select' => true,
+    ));
+    
+    // ...
+}
+```
+
+#### Advanced example
+
+The Bootstrap modal example:
+
+``` php
+public function buildDatatable(array $options = array())
+{
+    // ...
+
+    $this->extensions->set(array(
+        'blurable' => false,
+        'className': 'selected',
+        'info': true,
+        'items': 'rows',
+        'selector': 'td, th',
+        'style': 'os',
+    ));
```

Comment:
I didn't work on others extensions, but if I compare this part with [buttons](https://github.com/stwe/DatatablesBundle/blob/master/Resources/doc/extensions.md#advanced-example) and [responsive](https://github.com/stwe/DatatablesBundle/blob/master/Resources/doc/extensions.md#advanced-example-1) extensions, options have to be embedded in an array with the name of the extension as the key of the array.

So this should like like:
```
$this->extensions->set(array(
    //'select' => true,
    'select' => array(
        'blurable' => false,
        'className': 'selected',
        'info': true,
        'items': 'rows',
        'selector': 'td, th',
        'style': 'os',
    ),
));
```

Reply:
OK, apparently I was sleeping while updating the documentation. I'll fix it.